src/components/FormLogin/index.test.tsx

```diff
@@ -1,18 +1,23 @@
 import { ChakraProvider } from '@chakra-ui/react';
-import { render, screen, within } from '@testing-library/react';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import { theme } from '@/theme/index';
 
+import { flashToastTestIds } from '../FlashToast/index';
 import FormLogin, { formLoginTestIds } from './index';
 
 describe('FormLogin', () => {
-  it('renders login form', () => {
+  const setup = () =>
```

Comment:
Can we change to use `beforeEach()` instead? 🤔

Reply:
@hoangmirs I try with that before but it didn't pass with the linter.
They suggest doing like this here.

<img width="500" alt="Screen Shot 2565-06-17 at 17 35 53" src="https://user-images.githubusercontent.com/749672/174282085-7a7f227c-28d8-496d-b906-c688aa454846.png">

https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-render-in-setup.md